docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -101,19 +103,20 @@ system's UNIX users.
 JupyterHub's [OAuthenticator][] currently supports the following
 popular services:
 
-- Auth0
-- Azure AD
-- Bitbucket
-- CILogon
-- GitHub
-- GitLab
-- Globus
-- Google
-- MediaWiki
-- Okpy
-- OpenShift
-
-A generic implementation, which you can use for OAuth authentication
+- [Auth0](https://github.com/jupyterhub/oauthenticator/blob/main/oauthenticator/auth0.py)
```

Comment:
There's a list of authenticators in the OAuthneticator docs here: https://oauthenticator.readthedocs.io/en/latest/api/index.html
For docs that aren't aimed at developers I think it's better to link to readable documentation instead of source code where reasonable.